source/releasenotes/2024-09-10-emergency-ticket-workspace-phase1.md

```diff
@@ -0,0 +1,18 @@
+---
+title: “On-call help” renamed to “Emergency ticket” (phase 1)
```

Comment:
Do we need "(phase 1)" at the end of the title? 

I understand that we're phasing the work, but could we say "… in the workspace dashboard" 

Alternately, should we wait to do the site and workspace dashboard change at the same time to avoid this confusion?

Reply:
It's good to separate the release notes for workspace and site level, considering customers can already see the changes on workspace level. What about I remove (phase 1) and replace it (workspace dashboard)? I think we need something to differentiate the 2 release notes.

Reply:
Ok, that works for me